### ISSUE TITLE
OSError: [E050] Can't find model 'en'. It doesn't seem to be a shortcut link ...

### ISSUE THREAD

**Issue by bhargavvader** [UChicago-CCA-2021/Frequently-Asked-Questions#6]
If you found yourself encountering this error, run ```python -m spacy download en``` in terminal and then run the command again.

**Comment by bhargavvader** [comment]
Check out this page for more details about what you are downloading: https://spacy.io/usage/models

Basically, think of them as these super lightweight neural based language models which perform a lot of the tasks which will be useful for us throughout the class.

**Comment by sabinahartnett** [comment]
Thanks! This solved that issue for me.

**Comment by chuqingzhao** [comment]
> If you found yourself encountering this error, run `python -m spacy download en` in terminal and then run the command again.

Thank you! I have tried to run 'python -m spacy download en' in terminal, but I met requests exceptions error: 
![image](https://user-images.githubusercontent.com/52275397/104947849-53a44200-59f7-11eb-8d02-1ee6a6759916.png)

**Comment by Rui-echo-Pan** [comment]
Same with @LouisaZhao1998 But I notice this is the problem with the connections, which I had met when requesting (scraping wiki), although I use VPN in China.
My past solution is to use the VPN from US university, and the connection issue can always be solved, same with this time.

Unfortunately, downloading is tooooo slow, and then encounter another problem:
<img width="624" alt="截屏2021-01-19 下午12 58 29" src="https://user-images.githubusercontent.com/52217605/104989751-0efbc380-5a56-11eb-883d-99ce121fe1e6.png">

Is there any solution?

**Comment by Rui-echo-Pan** [comment]
Yeah, I guess I solved it. The problems above may only occur for those in China, something related to the **Internet Service Provider**. 
This should be solved by using China Unicom Provider and a university VPN. China Mobile Provider always have the connection problem.

**Comment by Rui-echo-Pan** [comment]
<img width="511" alt="截屏2021-01-19 下午1 43 21" src="https://user-images.githubusercontent.com/52217605/104992568-4e2d1300-5a5c-11eb-8360-cf4b882a0d33.png">
Got solved! It's still due to the Internet connection problem. It's easily solved by changing to a better environment.

**Comment by chuqingzhao** [comment]
Thanks @Rui-echo-Pan . I can download the packages now, but when building dependencies, it shows another error "Could not build wheels for blis which use PEP 517 and cannot be installed directly". 
![image](https://user-images.githubusercontent.com/52275397/105004485-0368c680-5a6f-11eb-87bf-b362b0ceeb34.png)

![image](https://user-images.githubusercontent.com/52275397/105004409-e59b6180-5a6e-11eb-8b10-f2ca579af088.png)

I run pip install --upgrade pip setuptools wheel. Though I also have downloaded pep517 and satisfied wheel, the error still comes up.
![image](https://user-images.githubusercontent.com/52275397/105005277-0dd79000-5a70-11eb-99be-ac0a3695269b.png)

Any solution to the wheel problem? Thanks in advance.

**Comment by bhargavvader** [comment]
Seems to be an issue with certain libraries for Windows and spaCy, a quick google search on errors to do with failed building wheels for blis turns up:

https://stackoverflow.com/questions/43370851/failed-building-wheel-for-spacy

https://github.com/explosion/spaCy/issues/3019

https://github.com/explosion/spaCy/issues/4581

https://stackoverflow.com/questions/56666186/error-complete-output-from-command-error-installing-spacy-using-pip

Seems to also maybe do with your python version and the compiler. Hopefully one of those links has the answer!
